Buildlease/Buildlease/ClientApp/src/components/main-page/SearchBar.tsx

```diff
@@ -4,22 +4,23 @@ import { NavMenu } from '../layout/NavMenu';
 import styles from './main_page.module.css'
 import { SearchRounded as SearchIcon} from "@material-ui/icons"
 
+import { Input } from 'antd';
+
+const { Search } = Input;
```

Comment:
maybe just use Input.Search instead of this line and Search?

Reply:
done